src/modules/curations/itemCuration/sagas.spec.ts

```diff
@@ -129,7 +129,20 @@ describe('when third party items were fetched', () => {
 
   describe('and some of the items were published', () => {
     beforeEach(() => {
-      items = [{ id: 'itemId1', isPublished: false } as Item, { id: 'itemId2', isPublished: true } as Item]
+      items = [
+        {
+          id: 'itemId1',
+          collectionId: thirdPartyCollection.id,
+          urn: 'urn:decentraland:mumbai:collections-thirdparty:thirdparty2:tercer-fiesta-2',
+          isPublished: false
+        } as Item,
+        {
+          id: 'itemId2',
+          collectionId: thirdPartyCollection.id,
+          urn: 'urn:decentraland:mumbai:collections-thirdparty:thirdparty2:tercer-fiesta-2',
```

Comment:
this urn is the same as the item above, is that correct?

Reply:
Yeah, the important field here is the `isPublished` one, that should be `false` for the one and `true` for the other. The URN will be the same for this case because the items belong to the same collection.